reviews/methods1.Rmd

```diff
@@ -0,0 +1,450 @@
+
+Preregistration does not give certainty about the theoretical risk, as the authors argue.
+It gives more certainty in most cases at best, but it allows others to evaluate it.
+This might sound like a subtle difference, but I think it matters.
+
+> Agreed.
+
+The authors might reply that the did not really mean 'minimize uncertainty about the theoretical risk' but mainly meant that readers can judge for themselves what the Type 1 error of a test is (based on their own assumptions about what the most appropriate test is).
+In that case, their argument does not differ from mine in Lakens (2019), and the authors should explain what their contribution adds.
+
+> We took considerable time to put what we meant on paper.
+> In the revision, we tried hard to be clearer that uncertainty always refers to the uncertainty of the reader.
+> However, being content with claiming that readers have to judge for themselves what the theoretical risk/severity of a test results in a weak argument for preregistration.
+> We go further and show that for all individual assessments, preregistration has a benefit and that if one is willing to assume some parameters, this benefit is striking.
+
+Point 2: It remains unconvincing why preregistration is useful for exploratory studies.
+The authors seem to argue that even in exploratory studies preregistration will remove *some* uncertainty about the ability of a test to falsify a prediction.
+That is true.
+However, they do not argue why it would do so to an extent that makes preregistration useful in real life. If an action I do reduces my uncertainty from 98% to 94%, it has relatively reduced my uncertainty.
+At the same time, it has done so in a manner that does not matter at all.
+My point here is that the error rate in exploratory research is inflated above alpha% to an unknown extent.
+A preregistration might reduce the inflation somewhat.
+But the rate is quickly too high so that I wouldn't trust the claim anyway.
+
+> Actually, we think this is one of the strengths of the paper.
+> With the mathematical analysis, we show exactly how much uncertainty, theoretical risk, and detectability result in how much confirmation, as seen in Figures 1 and 2.
+> The vague intuition of a "[Type 1 error] rate [that] is quickly too high so that I wouldn't trust the claim anyway" is replaced by a clear functional form that enables navigating tradeoffs involving exploration, post-hoc changes or detectability.
+
+If a claim is made with 10% lower Type 1 error rate than it would have been made when it was not preregistered is not useful - because I do not know what the error rate associated with the claim is.
+
+> Our claim is that uncertainty is reduced, not theoretical risk directly.
+> We claim that there are exploratory methods with a well-known error rate.
+> For example, a neural network has extreme flexibility in fitting the data closely, yet it is calculatable.
```

Comment:
```suggestion
> For example, a neural network has extreme flexibility in fitting the data closely, yet it is calculable.
```

But how? Can you add a reference? Do you mean V-C dimension or something related?